CMakeLists.txt

```diff
@@ -40,6 +42,14 @@ add_library(rtcdcpp SHARED
         ${LIB_HEADERS}
         ${LIB_SOURCES})
 
+if (SPDLOG_FOUND)
```

Comment:
Is there a way to make this an explicit option, rather than implicit?

I think by _default_, most new people using the library are going to want some kind of logging, and should either install spdlog while they're getting started or explicitly turn it off if they want to build their own.

What do you think?